tslib/src/caching/redis-service.ts

```diff
@@ -41,26 +72,77 @@ export class RedisService implements IRedisService {
       }
     }
     this._redis = new Redis(options)
+    this._logger = new Logger.ConsoleLogger()
   }
 
   async get (key: string): Promise<any | null> {
```

Comment:
Needs Jsdoc.  Please mention it throws.

Reply:
i documented the interface given that it's consumed, whereas the implementation is merely injected

Reply:
I'm referring to the `get` method.

Reply:
`get` method on `IRedisService` interface updated with `@throws`

Reply:
would you prefer these doc attributes on the implementation, given that behaviour across implementations can differ?  
there is an argument for the interface, so you get enhanced intellisense while coding